src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/classes/eltwise_chain.cpp

```diff
@@ -135,6 +135,19 @@ void EltwiseChainTest::SetUp() {
 
         ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwiseOps[eltwiseOps.size() - 1])};
         function = std::make_shared<ov::Model>(results, paramVec, "eltwise_chain_fq");
+    } else if (postNode == PostNode::SOFTSIGN) {
+        std::vector<std::shared_ptr<ov::Node>> eltwiseOps;
+        eltwiseOps.push_back(make_eltwise(inputNodes1[0], inputNodes2[0], eltwiseOpTypes[0]));
+        for (size_t i = 1; i < eltwiseOpTypes.size(); i++) {
+            eltwiseOps.push_back(make_eltwise(eltwiseOps[eltwiseOps.size() - 1], inputNodes2[i], eltwiseOpTypes[i]));
+        }
```

Comment:
This part is duplicated in all 3 if-else branches. Can we move it before if?

Reply:
Quantize case differs `eltwiseOpTypes.size() - 1`
I moved vector initialization.